tests/ansible-navigator.yml

```diff
@@ -0,0 +1,16 @@
+---
+ansible-navigator:
```

Comment:
Should we be mashing up the python project tests with the container tests? It might be hard to identify what files are for what...

Reply:
I agree, but i will do the split after.